content/bn/docs/concepts/cluster-administration/networking.md

```diff
@@ -0,0 +1,81 @@
+---
+reviewers:
+- thockin
+title: ক্লাস্টার নেটওয়ার্কিং
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+নেটওয়ার্কিং কুবারনেটিসের একটি কেন্দ্রীয় অংশ,
+কিন্তু এটি কীভাবে কাজ করে তা বোঝা চ্যালেঞ্জিং হতে পারে। 
+মোট ৪টি আলাদা নেটওয়ার্কিং সমস্যা রয়েছে যা সমাধান করতে হয়:
+
+1. অত্যন্ত ঘনিষ্ঠভাবে সংযুক্ত কন্টেইনার-টু-কন্টেইনার যোগাযোগ: এটি {{< glossary_tooltip text="পড" term_id="pod" >}} এবং `localhost` যোগাযোগের মাধ্যমে সমাধান করা হয়।
+2. পড-টু-পড যোগাযোগ: এটি এই ডকুমেন্টের প্রধান আলোচ্য বিষয়।
+3. পড-টু-সার্ভিস যোগাযোগ: এটি [সার্ভিস](/docs/concepts/services-networking/service/) দ্বারা আলোচিত হয়েছে।
+4. বাহ্যিক-টু-সার্ভিস যোগাযোগ: এটিও সার্ভিস দ্বারা আলোচিত হয়েছে।
+
+<!-- body -->
+
+
+কুবারনেটিস মূলত অ্যাপ্লিকেশনগুলির মধ্যে মেশিন ভাগ করে নেওয়ার বিষয়ে।
+সাধারণত, মেশিন ভাগ করে নেওয়ার জন্য নিশ্চিত করতে হয় যে দুটি অ্যাপ্লিকেশন 
+একই পোর্ট ব্যবহার করার চেষ্টা না করে। বড় আকারের সিস্টেমে বিভিন্ন ডেভেলপারদের মধ্যে 
+পোর্ট সমন্বয় করা খুব কঠিন এবং ব্যবহারকারীদের নিয়ন্ত্রণের বাইরে ক্লাস্টার-লেভেলের সমস্যাগুলি প্রকাশ করে।
+
+ডাইনামিক পোর্ট বরাদ্দ সিস্টেমে অনেক জটিলতা আনে - প্রতিটি 
+অ্যাপ্লিকেশনকে পোর্টগুলি ফ্ল্যাগ হিসাবে নিতে হয়, API সার্ভারগুলিকে 
+কনফিগারেশন ব্লকগুলিতে ডাইনামিক পোর্ট নম্বর কীভাবে ঢোকাতে হয় তা জানতে হয়, 
+সার্ভিসগুলিকে একে অপরকে কীভাবে খুঁজে পেতে হয় তা জানতে হয়, ইত্যাদি। 
+এই সমস্যাগুলি সমাধান করার পরিবর্তে, কুবারনেটিস একটি ভিন্ন পদ্ধতি গ্রহণ করে।
+
+কুবারনেটিস নেটওয়ার্কিং মডেল সম্পর্কে জানতে, [এখানে](/docs/concepts/services-networking/) দেখুন।
+
+## কুবারনেটিস IP অ্যাড্রেস রেঞ্জ
+
+কুবারনেটিস ক্লাস্টারগুলির জন্য পড, সার্ভিস এবং নোডের জন্য অ-ওভারল্যাপিং IP অ্যাড্রেস বরাদ্দ করতে হয়, 
```

Comment:
Kindly use `নন-ওভারল্যাপিং` instead of `অ-ওভারল্যাপিং`.
কুবারনেটিস ক্লাস্টারগুলির জন্য পড, সার্ভিস এবং নোডের জন্য নন-ওভারল্যাপিং IP অ্যাড্রেস বরাদ্দ করতে হয়,